docs/howto/openstack/keystone/app-creds.md

```diff
@@ -0,0 +1,277 @@
+| name         | elated_jones                                             |
+| project_id   | dfc700467396428bacba4376e72cc3e9                         |
+| roles        | creator reader swiftoperator load-balancer_member member |
+| secret       | afduKgbMoqvf4UXyvYJAsWXN                                 |
+| system       | None                                                     |
+| unrestricted | False                                                    |
+| user_id      | e50719d594f84f16ad13f88da540f762                         |
++--------------+----------------------------------------------------------+
+```
+
+When creating new application credentials, you may optionally provide a description.
+More importantly, maybe you will want to set an expiration date too.
+See the example below, where we provide a description and, at the same time, set an expiration date:
+
+```console
+$ openstack application credential create \
+    --description 'Direct connection to Iqaluit' \
+    --expiration '2024-08-31T23:59:59' \
+    -f yaml goofy_dijkstra
+
+description: Direct connection to Iqaluit
+expires_at: '2024-08-31T23:59:59.000000'
+id: 43649f148bbe46ab8f1d745eea095db9
+name: goofy_dijkstra
+project_id: dfc700467396428bacba4376e72cc3e9
+roles: swiftoperator load-balancer_member reader creator member
+secret: XS7YUGrH_FGZG9cEjT75tEYB7ciu-3YHgCWOoguTV3sezww8n8xcq67ae_4phzoL_vLY2ryp6avcy6eG8ihiMA
+system: null
+unrestricted: false
+user_id: e50719d594f84f16ad13f88da540f762
+```
+
+Note that we opted for a YAML-formatted output, for this format makes it very easy to select the secret and copy it.
+
+## Listing and viewing application credentials
+
+To list all application credentials you have created, type the following:
+
+```console
+$ openstack application credential list
+
++-------------------+----------------+-------------------+-------------------+---------------------+
+| ID                | Name           | Project ID        | Description       | Expires At          |
++-------------------+----------------+-------------------+-------------------+---------------------+
+| 98a2804b2d294b678 | angry_lamarr   | dfc700467396428ba | None              | None                |
+| 4ae30ec67697382   |                | cba4376e72cc3e9   |                   |                     |
+| a39676b7b0bd4c0c8 | elated_jones   | dfc700467396428ba | None              | None                |
+| 4fa06bdde44b090   |                | cba4376e72cc3e9   |                   |                     |
+| 43649f148bbe46ab8 | goofy_dijkstra | dfc700467396428ba | Direct connection | 2024-08-            |
+| f1d745eea095db9   |                | cba4376e72cc3e9   | to Iqaluit        | 31T23:59:59.000000  |
++-------------------+----------------+-------------------+-------------------+---------------------+
+```
+
+You may select specific columns...
+
+```console
+$ openstack application credential list -c ID -c Name -c Expires\ At
```

Comment:
Somehow, I find `-c "Expires At"` (with quotes) more readable than the backslash escape.